Statut HSWro.tex

```diff
@@ -41,54 +41,52 @@ \section{CZŁONKOWIE}
         \item zwyczajnych,
         \item honorowych.
       \end{enumerate}
-    \item Członkiem Zwyczajnym Stowarzyszenia może być osoba fizyczna, która spełnia stosowne postanowienia ustawy Prawo o Stowarzyszeniach oraz wszystkie poniższe warunki:
+    \item Członkiem Zwyczajnym Stowarzyszenia może być osoba fizyczna, która spełni wszystkie poniższe warunki:
       \begin{enumerate}
+         \item przedstawi pozytywną opinię co najmniej trzech członków zwyczajnego Stowarzyszenia,
         \item złoży deklarację członkowską na piśmie,
```

Comment:
Po co to?

Reply:
tak jak masz 3 okejki obecnie tak masz to

Reply:
Raczej chodziło mi "...na piśmie...", ale ogarnąłem, że jakoś musimy to dokumentować.